modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/utils/constants.ts

```diff
@@ -23,6 +23,12 @@ const FUZZY_OPTIONS = {
 	pre: '<strong>',
 };
 
+const HEADERS = {
```

Comment:
To be more specific, maybe something like `OBJECTS_FETCH_HEADERS` or `DEFAULT_FETCH_HEADERS`?